src/hooks/useContractMethods.ts

```diff
@@ -0,0 +1,11 @@
+import { useActiveProvider } from './useActiveProvider'
+import { useContract } from './useContract'
+import { FOX_ETH_FARMING_ADDRESS } from 'lib/constants'
+import farmAbi from 'abis/farmingAbi.json'
+
+export const useContractMethods = (contractAddress?: string) => {
```

Comment:
should you allow passing in the abi as well to make this a more generic hook for using contract methods?

Reply:
Do the farming contracts have different abi's? Should I update the stakingContracts constant to include an abi as well?

Reply:
currently no, the v2 will have the same abi. but when reading the name of the hook it seemed to be a generic hook for any contract methods, i guess thats what led to me making that comment

Reply:
Oh ok yea I dont know if we actually even need that anymore any way I might just get rid of it.

Reply:
Updated